TeslaApp/TeslaApp/WelcomeView.swift

```diff
@@ -0,0 +1,180 @@
+            } label: {
+                LinearGradient.customLinearGradient
+                    .mask(
+                        Image(
+                            systemName: welcomeViewModel.isCarClose ? Constants.lockImageName : Constants
+                                .lockOpenImageName
+                        )
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                    )
+                    .neumorphismUnSelectedCircleStyle()
+            }
+        }
+        .padding(.vertical, 20)
+        .padding(.horizontal, 28)
+        .background(
+            RoundedRectangle(cornerRadius: 50)
+                .fill(
+                    welcomeViewModel.isCarClose ? LinearGradient(
+                        colors: [.backgroundCustom, .darkShadow.opacity(0.5)],
+                        startPoint: .bottomTrailing,
+                        endPoint: .topLeading
+                    ) : LinearGradient(
+                        colors: [.backgroundCustom, .darkShadow.opacity(0.5)],
+                        startPoint: .topLeading,
+                        endPoint: .bottomTrailing
+                    )
+                )
+        )
+    }
+
+    private var gradientBackground: some View {
+        Rectangle()
+            .fill(
+                LinearGradient(
+                    colors: [
+                        .backgroundCustom,
+                        .darkShadow,
+                        .darkShadow,
+                        .darkShadow,
+                        .backgroundCustom
+                    ],
+                    startPoint: .bottom,
+                    endPoint: .top
+                )
+            )
+            .frame(maxWidth: .infinity, maxHeight: .infinity)
+            .edgesIgnoringSafeArea(.all)
+            .opacity(welcomeViewModel.isCarClose ? 1 : 0)
+    }
+
+    private var standartGradientBackground: some View {
+        ZStack {
+            Rectangle()
+                .fill(
+                    LinearGradient(
+                        colors: [
+                            .backgroundCustom,
+                            .darkShadow
+                        ],
+                        startPoint: .top,
+                        endPoint: .bottom
+                    )
+                )
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+                .edgesIgnoringSafeArea(.all)
+            VStack {
+                Circle()
+                    .fill(LinearGradient.customLinearGradient)
+                    .frame(width: 100, height: 100)
+                    .shadow(color: .blueCustom, radius: 30, x: 0, y: 0)
+                    .blur(radius: 150)
+            }
+            .padding(.top, -100)
+        }
+        .opacity(welcomeViewModel.isCarClose ? 0 : 1)
+    }
+}
+
+struct WelcomeView_Previews: PreviewProvider {
+    static var previews: some View {
+        WelcomeView()
+            .previewDevice("iPhone 13")
```

Comment:
```suggestion
            .previewDevice("iPhone 13")
```
не уверен нужно ли в константы, но на всякий случай